[📝 Documentation] Add missing parameters from base.py (#4676)
GeotrekCE/Geotrek-admin: docs/advanced-configuration/attachments.rst

.. note:: 
  These three settings will also not allow downloading images from the parsers.

Paperclip enable video 
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> To enable the addition of a link to a YouTube/video in the attached files of objects:

Reviewer: ```suggestion
To enable adding YouTube or video links to the attached files of objects:
```
